addons/website/static/src/snippets/s_countdown/000.scss

```diff
@@ -0,0 +1,7 @@
+// s_countdown
+.s_countdown_enable_preview {
+    display: initial !important;
+}
+.s_countdown_none {
+    display: none !important;
+}
```

Comment:
This should not be in a 000.scss file, those are dedicated to style the snippet for the visitors and support snippet versioning. Those classes here are edit mode classes, that the visitors should thus not have to load. They also do not need versionning as our options are only meant to work on the last version of snippets.

You can thus put those in website.edit_mode.scss, no need of a new file. Besides, even if that was the correct way to go, your diff would not be ok as it is not a stable fix: you modify the XML assets, meaning users would have to do a -u, which they don't in stable versions. So, here someone getting your fix would only receive the JS part, toggling classes which have no effect (you would thus worsen the bug).